### DISCUSSION TITLE
Is there a public roadmap beyond the milestones ?

### DISCUSSION THREAD

**Discussion by EricBoix** [iTowns/itowns#2025] [General]
## Your Environment
Environment is not relevant.

## Context
Not relevant.

The [itowns milestones](https://github.com/iTowns/itowns/milestones) express (are a form of) a short term roadmap with an horizon of one to two months (two minor releases of one-month development cycles). Does there exist a mid-term roadmap and is it public ?

**Comment by jailln** [comment]
Hi @EricBoix, 
The milestones are not up to date but we plan to use them better in the coming weeks to give a more precise short-term vision on the tickets we plan to develop in the next versions.
Moreover, we are currently re-designing the website where there will be such a mid-term roadmap. It should be up online in the coming month hopefully :)

**Comment by EricBoix** [comment]
Thanks @jailln . IMHO the mid-term visibility of the roadmap is a key element for a community member that needs to organize/schedule its own developments. And this information might even be important for new community members that might be "torn-apart" when having to choose between iTowns and, its fork, [giro3d](https://gitlab.com/giro3d/giro3d).
So I'll wait and see.
